test/Match.test.js

```diff
@@ -19,7 +19,7 @@ const fsStub = sinon.stub(fs, 'readFileSync', () => {
 
 test.after('cleanup', t => fsStub.restore());
 
-test('searchString match should return correct details', async t => {
+test.serial('searchString match should return correct details', async t => {
```

Comment:
same as comments above it feels like changing to `.serial` to add `.plan()` can just be done without these and using the `end` event